### PR TITLE
4.7.6 - v4 Migration Cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR is part of issue #83 

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix liniting issues
- Fix dead links

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.6_Testing_for_Logout_Functionality_OTG-SESS-006.md [hunk 1]
```diff
@@ -19,52 +20,53 @@ Users of web browsers often don't mind that an application is still open and jus
...
------------
+## How to Test
+
+### Testing for Log Out User Interface:
 
-**Testing for log out user interface:**\
 Verify the appearance and visibility of the log out functionality in the user interface. For this purpose, view each page from the perspective of a user who has the intention to log out from the web application.
 
-**Result Expected:**\
+#### Result Expected:
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.6_Testing_for_Logout_Functionality_OTG-SESS-006.md]: Remove all Results Expected sections as they are not needed. They are still part of that same subsection. @kingthorin what do you think about this?

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.6_Testing_for_Logout_Functionality_OTG-SESS-006.md [hunk 2]
```diff
@@ -19,52 +20,53 @@ Users of web browsers often don't mind that an application is still open and jus
...
 
--   “Burp Suite - Repeater” - <http://portswigger.net/burp/repeater.html>
+It is expected that the invocation of a log out function in a web application connected to a SSO system or in the SSO system itself causes global termination of all sessions. An authentication of the user should be required to gain access to the application after log out in the SSO system and connected application.
 
-References
-----------
+## Tools
 
-**Whitepapers**
+- [Burp Suite - Repeater](http://portswigger.net/burp/repeater.html)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.6_Testing_for_Logout_Functionality_OTG-SESS-006.md]: Wrong link

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.6_Testing_for_Logout_Functionality_OTG-SESS-006.md [hunk 3]
```diff
@@ -19,52 +20,53 @@ Users of web browsers often don't mind that an application is still open and jus
...
 
--   “The FormsAuthentication.SignOut method does not prevent cookie reply attacks in ASP.NET applications” - <https://support.microsoft.com/en-us/kb/900111>
--   “Cookie replay attacks in ASP.NET when using forms authentication” - <https://www.vanstechelman.eu/content/cookie-replay-attacks-in-aspnet-when-using-forms-authentication>
+## References
 
-\
+### Whitepapers
 
+- [“Cookie replay attacks in ASP.NET when using forms authentication”](https://www.vanstechelman.eu/content/cookie-replay-attacks-in-aspnet-when-using-forms-authentication)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.6_Testing_for_Logout_Functionality_OTG-SESS-006.md]: Remove the double quotes surrounding the text

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.6_Testing_for_Logout_Functionality_OTG-SESS-006.md (hunk 1 above)
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.6_Testing_for_Logout_Functionality_OTG-SESS-006.md]: @ThunderSon Updated with fix

**Pull request merged**